json/src/main/java/io/airlift/json/ObjectMapperProvider.java

```diff
@@ -118,7 +118,7 @@ public ObjectMapper get()
         objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
 
         // skip fields that are null instead of writing an explicit json null value
```

Comment:
Maybe mention that this only applies to JSON object fields and specifically not maps or arrays.